compose/tox.yml

```diff
@@ -8,7 +8,7 @@ services:
   tox:
     build:
       context: ..
-      dockerfile: ./Dockerfile
+      dockerfile: ./compose/development.Dockerfile
```

Comment:
isn't it better to execute tox using the production enviroment (alpine) rather that the dev enviorment? I was only thinking on the dev enviroment (debian) for devserver / debug puproses

Reply:
Ah, OK. Yes, we can do that. Will fix.

Reply:
And, done in ea2a759